template/en/default/global/header.html.tmpl

```diff
@@ -315,7 +315,7 @@
                 <li><a href="user_profile">My Profile</a></li>
                 <li><a href="page.cgi?id=user_activity.html&amp;action=run&amp;who=[% user.login FILTER uri %]">My Activity</a></li>
                 <li><a href="request.cgi?requester=[% user.login FILTER uri %]&amp;requestee=[% user.login FILTER uri %]&amp;do_union=1&amp;group=type&amp;action=queue">My Requests</a></li>
-                <li><a href="userprefs.cgi">Preferences</a></li>
+                <li><a href="userprefs.cgi">Edit Profile & Preferences</a></li>
```

Comment:
I think this makes the menu too wide. Probably leave it as "Preferences".
The edit button makes it discoverable enough.